binaries.esy.json

```diff
@@ -1,10 +1,9 @@
 {
   "dependencies": {
-    "ocaml": ">= 4.2.0",
+    "ocaml": ">= 4.4.0",
```

Comment:
Is this properly a constraint of Bisect? Is there anything added by this PR to Bisect itself, that requires OCaml 4.4.0? The only candidate I see are the changes to the labeled argument representation.

If not, I think we should rely on ppxlib's constraints.

On the subject of those, I would weakly prefer if ppxlib supported 4.02. It's probably not so critical for Bisect, as a typical project can just run coverage on a newer version of the compiler.

Reply:
Yes, you're right, this is not needed here. I've gone back to the 4.2 bound.

I will mention that you'd prefer a 4.2 support for ppxlib in the next ppxlib meeting. But we've already talked about supporting 4.2 once and, I think, unless it's necessary, it's likely that we'll have to focus on other things.